### ISSUE TITLE
The egg of mortal essence as warlock

### ISSUE THREAD

**Issue by excentric21** [SunwellTracker/issues#2552]
Description: The egg of mortal essence is meant to be proccing with warlock abilities like life tap 

How it works: the egg of mortal essence is not proccing off warlock spells

How it should work: The egg of mortal essence is meant to be proccing with warlock abilities like life tap

Source (you should point out proofs of your report, please give us some source): https://www.youtube.com/watch?v=LSmDEKxPeO8

**Comment by kodolol** [comment]
**more proof:**

- https://www.mmo-champion.com/threads/658602-The-Egg-of-Mortal-Essence

> only procced from using Life Tap, it took about 7 casts to make it proc on occasions, even giving time of 1 minute for cooldown to go.

- https://www.wowhead.com/item=40685/the-egg-of-mortal-essence#comments

![image](https://user-images.githubusercontent.com/37703096/69462028-9ed5ad00-0d80-11ea-9bc0-7878b810267f.png)

**Comment by Azathothh** [comment]
![obraz](https://user-images.githubusercontent.com/37483678/69904611-9f85c900-13a8-11ea-9de9-b6a7ed194f91.png)

**Comment by DollpartsFTP** [comment]
So, you really not gonna realise you're playing a 3.3.5a patch server, linking sources from 3.0.9 and 3.1.3, and a video from opening tier retail WOTLK?

A quick search of patch notes showed that things like this were fixed even pre ToC, nevermind 3.3.5

**Comment by kodolol** [comment]
> So, you really not gonna realise you're playing a 3.3.5a patch server, linking sources from 3.0.9 and 3.1.3, and a video from opening tier retail WOTLK?
> 
> A quick search of patch notes showed that things like this were fixed even pre ToC, nevermind 3.3.5

What do you mean?

**Comment by Azathothh** [comment]
He mean that it shouldn't proc for warlocks, and he's right. Proofs used in this topic are from early wotlk when it worked for warlocks, but later it was corrected. We've overlooked this information and added proc on Life Tap, but it would be reverted.

https://www.mmo-champion.com/threads/658602-The-Egg-of-Mortal-Essence
https://wowwiki.fandom.com/wiki/The_Egg_of_Mortal_Essence

 2009-07-26, 12:32 AM 
"not a special post on the subject but for the purpose of any warlock that wants to search about this trinket, i did a small test and found:
*does not work on Fel Armor heal ticks
*does not work from Siphon Life heals
*does not work from Haunt heal
*although didn't test enough for Drain Life it did not proc
only procced from using Life Tap, it took about 7 casts to make it proc on occasions, even giving time of 1 minute for cooldown to go."

Patch 3.3.0 (08-Dec-2009): This item can no longer be triggered by Life Tap.
